skyportal/handlers/api/source.py

```diff
@@ -1298,6 +1336,45 @@ class Validator(Schema):
             query_results["sources"] = obj_list
 
         query_results = recursive_to_dict(query_results)
+        if includeGeojson:
+            features = []
+
+            # currently necessary for the sources to render
+            features.append(
+                Feature(
+                    geometry=Point([float(-180), float(-180)]),
+                    properties={"name": "tmp"},
+                )
+            )
+            features.append(
+                Feature(
+                    geometry=Point([float(-180), float(-180)]),
+                    properties={"name": "tmp"},
+                )
+            )
+            features.append(
+                Feature(
+                    geometry=Point([float(-180), float(-180)]),
+                    properties={"name": "tmp"},
+                )
+            )
+
+            for source in query_results["sources"]:
+                point = Point((source["ra"], source["dec"]))
+                if source["alias"] is not None:
+                    feat = Feature(
+                        geometry=point, properties={"name": ",".join(source["alias"])}
```

Comment:
I suggest just defining the name in these if else blocks, then afterwards use its value in a single Feature instantiation to avoid repeated code

Reply:
much better!